network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/spotlight_posts.html

```diff
@@ -0,0 +1,24 @@
+{% load wagtailimages_tags i18n %}
+
+<div class="row">
+
+  <div class="col-12 col-md-6">
+    <span class="bubble">{% trans "spotlight" %}</span>
+    {% image page.spotlight_image width-540 %}
```

Comment:
If someone uploads a _long_ image this could scroll down for a while. Seems unlikely, but using `{% image page.spotlight_image fill-540xXXX %}` could prevent this while maintaining a 540px width

Reply:
this is not the real frontend work, this is purely the stub.